### ISSUE TITLE
Pronounce open little windows and no sound occer in Edge browser.

### ISSUE THREAD

**Issue by liserjrqlxue** [sapjax/word-hunter#3]
I install this extension directly to Edge.
When press :sound: icon, it open a little windows (may be a web page with [`extension://`](extension://) as url prefix) for a while and quickly close it (may be the duration of the sound).

**Comment by sapjax** [comment]
> I install this extension directly to Edge. When press 🔉 icon, it open a little windows (may be a web page with `extension://` as url prefix) for a while and quickly close it (may be the duration of the sound).

This is actually how the pronunciation works, I create a window in the background, then create a `new Audio(src)` in the background page to make it sound.
```ts
const createAudioWindow = async (audio: string) => {
  let url = chrome.runtime.getURL('audio.html')
  url = `${url}?audio=${encodeURIComponent(audio)}`
  await chrome.windows.create({
    type: 'popup',
    focused: false,
    top: 1,
    left: 1,
    height: 1,
    width: 1,
    url
  })
}
```

It's a 1x1 px window at the left/top side of the screen, really hard to find it.
If there's a better way to play audio, I'll change the implementation.

**Comment by sapjax** [comment]
I found there is an API [chrome.offscreen](https://developer.chrome.com/docs/extensions/reference/offscreen/) exist in Chrome 109+.
I'll use it in the next version, therefor should no window appears anymore.

**Comment by sapjax** [comment]
already fixed in version 1.0.19.
